packages/teleport-plugin-next-context/src/index.ts

```diff
@@ -0,0 +1,95 @@
+import {
```

Comment:
this file is for context providers, not context declaration or read. I was expecting a context read/access not provision in this file. Maybe naming could be better.